handlers/pageData.go

```diff
@@ -201,3 +208,252 @@ func purgeAllSessionsForUser(ctx context.Context, userId uint64) error {
 	return err
 
 }
+
+func createMenuItems(active string, isMain bool) []types.MainMenuItem {
+	hiddenFor := []string{"confirmation", "login", "register"}
+
+	if utils.SliceContains(hiddenFor, active) {
+		return []types.MainMenuItem{}
+	}
+	return []types.MainMenuItem{
+		{
+			Label:    "Blockchain",
+			IsActive: active == "blockchain",
+			Groups: []types.NavigationGroup{
+				{
+					Links: []types.NavigationLink{
+						{
+							Label: "Epochs",
+							Path:  "/epochs",
+							Icon:  "fa-history",
+						},
+						{
+							Label: "Slots",
+							Path:  "/slots",
+							Icon:  "fa-cube",
+						},
+					},
+				}, {
+					Links: []types.NavigationLink{
+						{
+							Label: "Blocks",
+							Path:  "/blocks",
+							Icon:  "fa-cubes",
+						},
+						{
+							Label: "Txs",
+							Path:  "/transactions",
+							Icon:  "fa-credit-card",
+						},
+						{
+							Label: "Mempool",
+							Path:  "/mempool",
+							Icon:  "fa-upload",
+						},
+					},
+				},
+			},
+		},
+		{
+			Label:    "Validators",
+			IsActive: active == "validators",
+			Groups: []types.NavigationGroup{
+				{
+					Links: []types.NavigationLink{
+						{
+							Label: "Overview",
+							Path:  "/validators",
+							Icon:  "fa-table",
+						},
+						{
+							Label: "Slashings",
+							Path:  "/validators/slashings",
+							Icon:  "fa-user-slash",
+						},
+					},
+				}, {
+					Links: []types.NavigationLink{
+						{
+							Label: "Validator Leaderboard",
+							Path:  "/validators/leaderboard",
+							Icon:  "fa-medal",
+						},
+						{
+							Label: "Deposit Leaderboard",
+							Path:  "/validators/deposit-leaderboard",
+							Icon:  "fa-file-import",
+						},
+					},
+				}, {
+					Links: []types.NavigationLink{
+						{
+							Label: "Deposits",
+							Path:  "/validators/deposits",
+							Icon:  "fa-file-signature",
+						},
+						{
+							Label: "Withdrawals",
+							Path:  "/validators/withdrawals",
+							Icon:  "fa-money-bill",
+						},
+					},
+				},
+			},
+		},
+		{
+			Label:    "Dashboard",
+			IsActive: active == "dashboard",
+			Path:     "/dashboard",
+		},
+		{
+			Label:    "Notifications",
+			IsActive: active == "notifications",
```

Comment:
In the original code, the `Notifications` entry was never highlighted. So this is a behavior change, how dare you?

Anyhow, I think it does not work like that. The `active` parameter seems to be set to `"user"` and not to `"notifications"`. Please check, see here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cb4c2faccaf0e8257685d3751a9fc3093612af7e/handlers/user.go#L616

Reply:
You are right as there are also other navigation links leading to user, I will remove the highlighting for the Notification tab again